schedule/sync-missing-okta-users.js

```diff
@@ -0,0 +1,31 @@
+import { Client } from '@okta/okta-sdk-nodejs'
+import rollbar from '../config/rollbar'
+import { SNS } from 'aws-sdk'
+
+export const handler = async (lambdaEvent) => {
+  const okta = new Client()
+  const sns = new SNS({ apiVersion: '2010-03-31', region: 'us-east-1' })
+  try {
+    const users = await okta.listGroupUsers(process.env.OKTA_MISSING_GROUP_ID, { limit: 100 })
+
+    await Promise.allSettled(users.map(user => {
+      return sns.publish({
+        TargetArn: process.env.SNS_OKTA_EVENTS_ARN,
+        Message: JSON.stringify({
+          eventType: 'user.account.update_profile',
+          target: [{ id: user.id }],
+          debugContext: { debugData: { changedAttributes: 'Notes' } }
```

Comment:
Is that a real attribute?

Reply:
if it's not, maybe a more obvious fake name would be better, like `dummy_to_trigger_change`)

Reply:
It's a real attribute, and it's the one I always fiddle with manually to trigger a sync in Okta UI.